hunts/chart_utils.py

```diff
@@ -0,0 +1,41 @@
+from django.utils import timezone
+
+from .models import Hunt
+from puzzles.models import Puzzle
+
+
```

Comment:
Another nit: both functions in this file are unit-testable. Could you add tests?

Reply:
Ah never mind I see that you already did. 👍

Reply:
Tests have been added!